routing/pathfind.go

```diff
@@ -40,6 +40,18 @@ type pathFinder = func(g *graphParams, r *RestrictParams,
 	source, target route.Vertex, amt lnwire.MilliSatoshi) (
 	[]*channeldb.ChannelEdgePolicy, error)
 
+var (
+	// DefaultPaymentAttemptPenalty is the virtual cost in path finding weight
+	// units of executing a payment attempt that fails. It is used to trade
+	// off potentially better routes against their probability of
+	// succeeding.
+	DefaultPaymentAttemptPenalty = lnwire.NewMSatFromSatoshis(100)
+
+	// DefaultMinProbability is the default minimum probability for routes
+	// returned from findPath.
+	DefaultMinProbability = float64(0.05)
```

Comment:
this value seems a little high to use an initial minimum, though I haven't worked with the PR as much as yourself. it might be useful to start low and then bump after we are confident in the behavior in the wild. mainly thinking about the situation where last resort paths exist, but mission control prevents us from exploring them

Reply:
Yes I wasn't sure about this value either. I will lower it to 1%. The worst that can happen is that a route is tried in a loop until the payment attempt timeout.

Reply:
sgtm